ee/modules/600-flant-integration/docs/README_RU.md

```diff
@@ -103,3 +93,98 @@ kubectl annotate node test pricing.flant.com/nodeType=Special
 | absent                | В кластере не хватает Node или NodeGroup.                                                 |
 | changed               | `terraform plan` предполагает изменение объектов в облаке без их удаления.                |
 | ok                    | Расхождений не обнаружено.                                                                |
+
+
+## Сбор данных
+
+### Куда Deckhouse отправляет данные?
+
+Все данные отправляются через единую точку входа. Этой точкой является сервис `connect.deckhouse.io` (далее — *Connect* или *сервис Connect*).
+
+Для авторизации при отправке данных каждый кластер Deckhouse отправляет ключ лицензии (license key) в качестве [Bearer Token](https://oauth.net/2/bearer-tokens/). Сервис Connect проверяет его на правильность, после чего в случае успеха перенаправляет запрос в необходимый внутренний сервис Флант. 
+
+Для отправки данных из кластера необходимо разрешить доступ до всех IP-адресов следующих DNS имен:
+- `сonnect.deckhouse.io`;
+- `madison-direct.deckhouse.io`.
+
+### Какие данные отправляет Deckhouse?
+
+> [Как отключить отправку данных Deckhouse...](#как-отключить-отправку-данных-deckhouse)
+
+Данные, отправляемые из каждого кластера:
+- Статистика о состоянии кластера: 
+  - версия Kubernetes;
+  - версия Deckhouse;
+  - канал обновлений; 
+  - количество узлов, и т.п.
+- Оповещения, отправляемые в систему работы с инцидентами Madison.
+- Метрики SLA по всем компонентам Deckhouse.
+- Логи оператора Deckhouse для отладки.
+- Способ подключения к master-узлам кластера.
+
+#### Статистика о состоянии кластера
+
+При помощи [shell-operator](https://github.com/flant/shell-operator) модуль flant-integration выполняет сбор метрик о состоянии объектов кластера. Затем с помощью [Grafana agent](https://github.com/grafana/agent) собранные метрики передаются по протоколу [Prometheus Remote Write](https://docs.sysdig.com/en/docs/installation/prometheus-remote-write/).
+
+На основании собранных данных:
+- Рассчитывается стоимость услуги [Managed Kubernetes](https://flant.ru/services/managed-kubernetes-as-a-service).
+- Происходит планирование миграций.
```

Comment:
> Происходит планирование миграций.

This doesn't make any sense

> Контролируется гомогенность инфраструктуры.

And this one, tool.

Reply:
Fixed

Reply:
Fixed